RxSwift/Observables/Implementations/DefaultIfEmpty.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  DefaultIfEmpty.swift
+//  Rx
+//
+//  Created by sergdort on 23/12/2016.
+//  Copyright © 2016 Krunoslav Zaher. All rights reserved.
+//
+
+import Foundation
+
+class DefaultIfEmptySink<SourceType, O: ObserverType>: Sink<O>, ObserverType where O.E == SourceType {
+    private let _defaultValue: SourceType
+    private var isEmpty = true
```

Comment:
_isEmpty :)

I know that we are using maybe a slightly weird convention, but it helps a lot when reasoning about local state of the operator.